ADVANCED.md

```diff
@@ -62,6 +62,29 @@ The link resolution is applied to one level deep by default. If you need it to b
 
 By default, the SDK will keep links, which could not get resolved, in your response. If you want to completely remove fields which could not be resolved, set `removeUnresolved: true` in the configuration options.
 
+### Links to other spaces (beta)
+
+`getEntry` and `getEntries` endponts can now resolve links to entries from other spaces as they do for regular links. The `resolveLinks`, `removeUnresolved` and `include` parameters affect both types of links.
+
+To use this functionality provide `additionalTokens` when creating the Contentful client
+```js
+const contentful = require("contentful");
+const client = contentful.createClient({
+  accessToken: "<your-access-token>",
+  space: "<your-space-id>",
+  additionalTokens: {
+    "<other-space-id>": "<other-access-token>",
```

Comment:
maybe make it clear that the access token is that space access token by rephrasing to `<other-space-access-token>`